ui/src/transfer.ts

```diff
@@ -57,7 +57,7 @@ const validateSufficientBalance = (fee: number, payerAccountId: string) => (
 ): Promise<boolean> => {
   return account
     .getBalance(payerAccountId)
-    .then(balance => balance >= +amount + fee);
+    .then(balance => currency.microRadToRad(balance) >= +amount + fee);
```

Comment:
I think it should be 
```suggestion
    .then(balance => balance >= currency.radToMicroRad(amount) + fee);
```

`balance` is obtained in `microRad`, `fee` is specified in `microRad` as well, only `amount` (specified by the user) is specified in Rad and thus needs to be converted to microRad here.

Reply:
I understood that the `fee` is defined in `SendFunds` as Rad (see [here](https://github.com/radicle-dev/radicle-upstream/pull/712/files#diff-2a6d3c83a0fe02f639bc62d73afa75dcR43) as it is then shown in the transaction view as `0.000001`

Reply:
hm no. it is defined as:

```
// TODO(nuno): receive this from the user once possible
const transactionFee = 1;
```

This is in microRad. It is then converted and shown in the UI as RAD, thus the `0.000001`.

Reply:
yes, I got confused. Fixed it in 6c70378